packages/toolpad-core/src/layout/DashboardLayout.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import {
+  AppBar,
+  Badge,
+  Box,
+  Container,
+  Divider,
+  Drawer,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  NoSsr,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+// Must use named imports or icons crash when the component is imported to a Next.js app
+// eslint-disable-next-line no-restricted-imports
+import {
+  Notifications as NotificationsIcon,
+  AccountCircle as AccountCircleIcon,
+  Mail as MailIcon,
+  Inbox as InboxIcon,
+} from '@mui/icons-material';
+
+const DRAWER_WIDTH = 240;
+
+interface DashboardLayoutProps {
+  children: React.ReactNode;
+}
+
+export default function DashboardLayout({ children }: DashboardLayoutProps) {
+  return (
+    <NoSsr>
```

Comment:
Theming isn't working without `NoSsr`, trying to understand these types of issues better there seem to be a lot of them...